src/omg-tf/err

```diff
@@ -0,0 +1,6 @@
+[31mTwo interrupts received. Exiting immediately. Note that data
```

Comment:
Was this mistakenly committed? If not, consider removing the ANSI escape codes for readability